src/components/BalanceList/BalanceListInterface.ts

```diff
@@ -0,0 +1,3 @@
+export default interface BalanceListInterface {
```

Comment:
for me extracting interface to a separate file does not make a lot of sense as it tightly coupled with the component. Why do you decide to extract it?

Reply:
maybe putting an interface in an `index.tsx` file inside component dir is a good idea. This way it will be _shared_ and easily imported from the top of component